core/serialization/workspaces.js

```diff
@@ -62,7 +62,8 @@ const load = function(state, workspace, {recordUndo = false} = {}) {
 
   const deserializers =
       Object.entries(serializerMap).sort(([, {priority: priorityA}], [
-                                           , {priority: priorityB}
+                                           ,
+                                           {priority: priorityB},
                                          ]) => priorityB - priorityA);
```

Comment:
Wow, that is… not pleasant looking.  I can't help thinking that this might be more readable the old-fashioned way:
```
   Object.entries(serializerMap).sort((a,b) => b[1].priority - a[1].priority);
```

Reply:
Yup, I find your version much easier to read. I assume that the :+1: means @BeksOmega is fine with it as well.